tokens.toml

```diff
@@ -0,0 +1,10 @@
+# Token accounts with their validity predicates
+
+[token.NAAN]
+denom = 6
+
+[token.NAM.masp_params]
+max_reward_rate = "0.0"
+kd_gain_nom = "0.1"
+kp_gain_nom = "0.1"
```

Comment:
these should be "0.25"

Reply:
Wdym? Are they not configurable?

Reply:
In principle it can be changed via governance, but the intention is to tune this for genesis, and 0.25 is the value that was determined to be desired for the inflation mechanism. 0.1 was an old placeholder value. Please change to 0.25.

Reply:
inflation should be 0 to begin with